orpheusdl/core.py

```diff
@@ -109,21 +132,34 @@ def __init__(self, private_mode=False):
                     raise Exception('Error loading extension: "{extension}"')
 
         # Module preparation (not loaded yet for performance purposes)
-        if not os.path.exists('modules'): os.makedirs('modules')
-        module_list = [module.lower() for module in os.listdir('modules') if os.path.exists(f'modules/{module}/interface.py')]
-        if not module_list or module_list == ['example']:
+
+        # Load modules from installed packages matching "orpheusdl_module_*"
+        module_packages = [
+            module_package for _, module_package, _ in pkgutil.iter_modules() if module_package.startswith('orpheusdl_module_')
```

Comment:
This is going to create a problem, because Pythons setuptools runs safe_name which will convert all `_` to `-`. So it should be `orpheusdl-module-`